services/src/test/java/org/jboss/windup/web/services/service/MigrationProjectServiceTest.java

```diff
@@ -0,0 +1,163 @@
+package org.jboss.windup.web.services.service;
+
+import org.jboss.windup.web.addons.websupport.WebPathUtil;
+import org.jboss.windup.web.services.model.*;
+import org.jboss.windup.web.services.model.Package;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import javax.persistence.EntityManager;
+import javax.persistence.TypedQuery;
+import javax.ws.rs.BadRequestException;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashSet;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+public class MigrationProjectServiceTest {
+    protected MigrationProjectService migrationProjectService;
+
+
+
+    @Before
+    public void setUp()
+    {
+        this.migrationProjectService = new MigrationProjectService();
+        this.migrationProjectService.entityManager = mock(EntityManager.class);
+        this.migrationProjectService.analysisContextService = mock(AnalysisContextService.class);
+        this.migrationProjectService.webPathUtil = mock(WebPathUtil.class);
+    }
+
+    @Rule
+    public ExpectedException thrown= ExpectedException.none();
+
+
+
+    @Test
+    public void testCreateProject()
+    {
+        MigrationProject project = new MigrationProject();
+        project.setId(new Long(1));
+        ArrayList projectList = new ArrayList();
+
+        TypedQuery mockQuery = mock(TypedQuery.class);
+        doReturn(mockQuery).when(this.migrationProjectService.entityManager).createQuery("SELECT proj FROM MigrationProject proj WHERE proj.title = :title and proj.provisional = FALSE", MigrationProject.class);
+        doReturn(mockQuery).when(mockQuery).setParameter("title", "testProject2");
```

Comment:
change to `doReturn(mockQuery).when(mockQuery).setParameter("title", project.getTitle());`
add `doReturn(mockQuery).when(mockQuery).setParameter("project", project);`